homeassistant/components/openuv/config_flow.py

```diff
@@ -10,42 +10,41 @@
     CONF_LATITUDE,
     CONF_LONGITUDE,
 )
-from homeassistant.core import callback
 from homeassistant.helpers import aiohttp_client, config_validation as cv
 
-from .const import DOMAIN
+from .const import DOMAIN  # pylint: disable=unused-import
 
 
-@callback
-def configured_instances(hass):
-    """Return a set of configured OpenUV instances."""
-    return {
-        f"{entry.data.get(CONF_LATITUDE, hass.config.latitude)}, "
-        f"{entry.data.get(CONF_LONGITUDE, hass.config.longitude)}"
-        for entry in hass.config_entries.async_entries(DOMAIN)
-    }
-
-
-@config_entries.HANDLERS.register(DOMAIN)
-class OpenUvFlowHandler(config_entries.ConfigFlow):
+class OpenUvFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle an OpenUV config flow."""
 
     VERSION = 2
     CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
 
-    async def _show_form(self, errors=None):
-        """Show the form to the user."""
-        data_schema = vol.Schema(
+    @property
+    def data_schema(self):
+        """Return the data schema."""
+        return vol.Schema(
             {
                 vol.Required(CONF_API_KEY): str,
-                vol.Optional(CONF_LATITUDE): cv.latitude,
-                vol.Optional(CONF_LONGITUDE): cv.longitude,
-                vol.Optional(CONF_ELEVATION): vol.Coerce(float),
+                vol.Optional(
+                    CONF_LATITUDE, default=self.hass.config.latitude
```

Comment:
The downside of filling it in as defaults during the making of a config entry is that it won't automatically update whenever the Home Assistant configuration updates.

Reply:
Ah, I see – so, the better road would be to not store the coordinates in the configure entry at all (and figure out whether to use the HASS values further downstream)?

Reply:
924a509. I'll implement home tracking in another PR.